docs/modules/maintain-cluster/pages/restart-cluster.adoc

```diff
@@ -0,0 +1,189 @@
+
+Persistence is required to avoid data loss during a whole cluster restart. See xref:storage:configuring-persistence.adoc[] for details.
+
+=== Embedded mode
+
+If your cluster is operating in embedded mode, you should restart your host application instead of trying to restart the embedded Hazelcast members independently.
+
+== Perform a restart
+
+This section describes how to restart a cluster using a choice of common tools. Hazelcast supports a wide range of deployment options that may require other tools or allow you to automate parts of the process, but the required steps will be similar.
+
+You should perform restarts in a low utilization or maintenance window to reduce the risk of service impacts.
+
+[[rolling]]
+=== Perform a rolling restart
+
+You should check the cluster is healthy before and after making changes. The best way to do this is to run a xref:{page-latest-supported-mc}@management-center:clusters:healthcheck.adoc[Healthcheck] in Management Center. For other options, see xref:maintain-cluster:monitoring.adoc[].
+
+To perform a rolling restart:
+
+. Update the static configuration as required.
+
+. Choose a member and make a note of its name or IP address to help you keep track.
+
+. Gracefully shut down the member.
++
+[tabs]
+====
+CLI::
++
+IMPORTANT: This command shuts down all running members on the local machine.
++
+--
+[source,bash]
+----
+hz-stop
+----
+--
+
+Management Center::
++
+--
+Go to *Members*, select the member's row in the table and select *Shutdown*.
+--
+
+Docker::
++
+--
+[source,bash]
+----
+docker stop <CONTAINER_NAME>
+----
+--
+====
+
+. Wait until all partition migrations are completed. You can monitor the progress of migrations on the xref:{page-latest-supported-mc}@management-center:clusters:dashboard.adoc[cluster dashboard] in Management Center.
+
+. Start a new member.
++
+[tabs]
+====
+CLI::
++
+--
+[source,bash]
+----
+hz start
+----
+--
+
+Docker::
++
+--
+[source,bash]
+----
+docker start <CONTAINER_NAME>
+----
+--
+====
+
+. Wait until the new member joins the cluster and all partition migrations are completed. Again, you can monitor progress using the xref:{page-latest-supported-mc}@management-center:clusters:dashboard.adoc[cluster dashboard] in Management Center.
+
+. Repeat this process until all members have been restarted with the updated configuration.
+
+[[whole]]
+=== Perform a whole cluster restart
+
+You should check the cluster is healthy before and after making changes. The best way to do this is to run a xref:{page-latest-supported-mc}@management-center:clusters:healthcheck.adoc[Healthcheck] in Management Center. For other options, see xref:maintain-cluster:monitoring.adoc[].
+
+To perform a whole cluster restart:
+
+. Update the static configuration as required.
+
+. Divert traffic to another Hazelcast cluster. How to do this depends on how your Hazelcast deployment is architected.
++
```

Comment:
IMO this looks a bit odd, single bullet point should be better:

<img width="425" height="110" alt="image" src="https://github.com/user-attachments/assets/cbce16dc-7d63-4183-9fb3-28311c4b55a3" />